packages/website/components/account/fileUploader/fileUploader.js

```diff
@@ -74,6 +77,20 @@ const FileUploader = ({ className = '', content, uploadModalState, background })
     [uploadsProgress]
   );
 
+  const acceptedTerms = e => {
+    setHasAcceptedTerms(e.target.checked);
+    if (e.target.checked) {
+      localStorage.setItem('acceptedTerms', Date.now().toString());
+    } else {
+      localStorage.removeItem('acceptedTerms');
+    }
+  };
+  useEffect(() => {
+    const acceptedTermsLocalStorage = localStorage.getItem('acceptedTerms');
+    console.log(acceptedTermsLocalStorage);
```

Comment:
should probably rm this `console.log` or add some context so it's not just a random timestamp or null